pkg/disttask/importinto/clean_up.go

```diff
@@ -48,7 +53,22 @@ func (*ImportCleanUpS3) CleanUp(ctx context.Context, task *proto.Task) error {
 		return err
 	}
 	defer redactSensitiveInfo(task, taskMeta)
-	// Not use cloud storage, no need to clean up.
+
+	if !taskMeta.ResetTableMode {
```

Comment:
is there need for this check, we can alter the table directly, `AlterTableMode` does nothing if it's already in `Normal`, we can skip changing the task meta